src/main/java/org/folio/rest/handlers/LoanDueDateChangedEventHandler.java

```diff
@@ -27,30 +20,9 @@ public LoanDueDateChangedEventHandler(PostgresClient postgresClient) {
 
   @Override
   public Future<String> handle(LoanDueDateChangedEvent event) {
-    return succeededFuture(event.getUserId())
-      .compose(userSummaryRepository::findByUserIdOrBuildNew)
-      .compose(summary -> updateUserSummary(summary, event))
+    return eventService.save(event)
+      .map(eventId -> event.getUserId())
```

Comment:
Redundant mapping?

Reply:
Why is it redundant? We're mapping from `eventId` to `userId`.

Reply:
But it the next line you name it `eventId` again and then never use it.

Reply:
I assume you intended to do something like this:
```
return eventService.save(event)
      .map(eventId -> event.getUserId())
      .compose(userSummaryService::rebuild)
      .onComplete(result -> logResult(result, event));
```